src/output/distWriter.ts

```diff
@@ -85,10 +85,11 @@ export function distWriter(props: IOuputParserProps): DistWriter {
       };
     },
     // the actual write function
-    write: async (target: string, contents: Buffer | string) => {
+    write: async (target: string, contents: Buffer | string): Promise<string> => {
       if (!path.isAbsolute(target)) target = path.join(self.outputDirectory, target);
       ensureDir(path.dirname(target));
-      writeFile(target, contents);
+      await writeFile(target, contents);
```

Comment:
add comment // @todo: try {} catch?

Reply:
I think this one needs to be caught at a higher level...

Reply:
Okay, let's figure out how we want to handle these later